docs/manuals/core/config/data-node-config.md

```diff
@@ -856,4 +858,32 @@ Then we configure 5 data nodes:
 - Lines 18 and 19 configure 2 SQL Table data nodes, one using `Config.configure_data_node()^` with `storage_type="sql_table"`, one using `Config.configure_sql_table_data_node()^`. Since both have the same _**storage_type**_ as the default data node configuration, both have the same properties except for the table name.
 - Line 21 configures a CSV data node `forecast_data_cfg`. Since the _**storage_type**_ is `"csv"`, which is different from the `"sql_table"` configured in line 9, the default data node configuration is ignored. Therefore, the scope of `forecast_data_cfg` is `SCENARIO` by default.
 
+# Configure a data node from another configuration
+
+Taipy also provides the possibility to use an existing configuration as a scaffold to configure a new data node.
+This can be useful when the application has a lot of data nodes with similar properties.
+
+To utilize the information of an existing configuration to create a new data node configuration, one can use the
+`Config.configure_data_node_from()^` method. This method accepts the following parameters:
+
+- _**source_configuration**_ is a mandatory parameter that represents the source data node configuration.
```

Comment:
_**source_configuration**_ is a mandatory parameter representing the source data node configuration.